pkgs/by-name/li/libwacom/package.nix

```diff
@@ -37,14 +37,18 @@ stdenv.mkDerivation (finalAttrs: {
     pkg-config
     meson
     ninja
-    python3
```

Comment:
Is it not needed at build time? Does cross compilation still work?

Reply:
You are right! It is needed.
I tried `nix-build -A pkgsCross.aarch64-multiplatform.libwacom`.
It fails without and succeeds with.